tests/homepage.spec.ts

```diff
@@ -31,18 +33,12 @@ test("schema is plumbed along to the sketch page", async ({ page }) => {
     .fill("LAD_Adur");
   await page.getByRole("button", { name: "Start" }).click();
   await expect(page).toHaveURL(
-    /.*scheme.html\?authority=LAD_Adur&schema=pipeline/,
+    /.*files.html\?authority=LAD_Adur&schema=pipeline/,
   );
-});
 
-test("a v1 file with a pipeline hint redirects to pipeline mode", async ({
```

Comment:
This behavior is removed; people can create a new file and add scheme from file instead